packages/storybook-vue/src/stories/Textarea/TextareaBinding.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Textarea } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-textarea-with-form-binding" title="Vue.js Component/Textarea" component={Textarea} argTypes={argTypes} />
+
+# Textarea
+
+export const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { Textarea },
+  setup() {
+    return { args };
+  },
+  template: `
+  <FormField> 
+    <Textarea v-bind="args" @update:modelValue="action" :value="args.modelValue" />
+  <br/>
+  <br/>
+  <Paragraph v-if="args.modelValue"><strong> Textarea Binding:</strong> {{args.modelValue}}</Paragraph>
+  </FormField>
```

Comment:
I think there are ways to use more semantic HTML for displaying the form binding.

Instead of inside `FormField`, we can use something like this after `<FormField>`:

```
<dl v-if="args.modelValue">
  <div aria-live="polite" aria-atomic="true">
    <dt>Textarea Binding:</dt>
    <dd>{{args.modelValue}}</dd>
  </div>
</dl>
```